### ISSUE TITLE
IE11 Issue

### ISSUE THREAD

**Issue by mrdigitalau** [olefirenko/vue-google-autocomplete#127]
Hey mate, thanks very much for this plugin.

Works great for me on most browsers but seems to cause an error on IE11 which stops vue from working completely.

I get an error in IE11: 

`SCRIPT1003: Expected ':' error on IE11`

And it specifically refers to this line in the code of your plugin:

`  this.$emit('inputChange', { newVal, oldVal }, this.id);`

I am using Babel but it doesn't seem to fix it..

Any ideas how to get around this?

Cheers

**Comment by jebarjonet** [comment]
The problem comes from here https://github.com/olefirenko/vue-google-autocomplete/blob/master/src/VueGoogleAutocomplete.vue#L70

Because IE11 does not interpret ES5 syntax correctly, instead of:
```js
data() { ... }
```
it wants
```js
data: function () { ... }
```
That's because this library is not compiled when distributed. This is related to https://github.com/olefirenko/vue-google-autocomplete/issues/92

**Comment by mrdigitalau** [comment]
Hey @jebarjonet, did you figure out a workaround on it by any chance? I am assuming we need to fork the project to fix it?

**Comment by jebarjonet** [comment]
@mrdigitalau yes, this library being only 1 component I copy/pasted it in my code directly and kept only the pieces I needed.
